### DISCUSSION TITLE
Batch update/insert for stores

### DISCUSSION THREAD

**Discussion by fullben** [JanWiemer/jacis#22] [Ideas]
Currently, new values can only be added to stores by calling the `update(K key, TV value)` method.

Would it be possible to implement a method along the signature of `void update(Function<TV, K> keyFunction, Collection<TV> value)` for batch inserts?

**Comment by JanWiemer** [comment]
Good idea! I added an issue for this: https://github.com/JanWiemer/jacis/issues/23#issue-986707680
I will try to do this in the near future...
